libs/core/langchain_core/utils/function_calling.py

```diff
@@ -371,9 +377,18 @@ def convert_to_openai_function(
             " 'title' and 'description' keys."
         )
 
+    if strict is not None:
+        oai_function["strict"] = strict
+        # As of 08/06/24, OpenAI requires that additionalProperties be supplied and set
+        # to False if strict is True.
+        oai_function["parameters"]["additionalProperties"] = False
+    return oai_function
+
 
 def convert_to_openai_tool(
     tool: Union[Dict[str, Any], Type[BaseModel], Callable, BaseTool],
+    *,
+    strict: Optional[bool] = None,
```

Comment:
What's the difference between None and False?

Reply:
ah i see below -- maybe we can expand doc-string here as well since some users use this function directly?